bessctl/commands.py

```diff
@@ -1532,11 +1532,12 @@ def tcpdump_module(cli, module_name, direction, gate, opts):
             pass
 
 
-@cmd('track ENABLE_DISABLE [MODULE] [DIRECTION] [GATE]',
+@cmd('track ENABLE_DISABLE [DIRECTION] [MODULE] [GATE]',
      'Count the packets and batches on a gate')
```

Comment:
The only argument I have for the reordering is that there is only two directions and there can be plenty of modules.  At any rate, I'm OK with not changing the order.  However, I think introducing * as a wildcard for modules would be simpler than using keyword arguments.  If you like the idea, I can implement it.